packages/FormElement/stories/examples/Showcase.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { boolean, select, text } from "@storybook/addon-knobs";
+import Heading from "@paprika/heading";
+import Input from "@paprika/input";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import { FormElementStory } from "../FormElement.stories.styles";
+import FormElement from "../../src";
+
+const formElementProps = () => ({
+  hasOptionalLabel: boolean("hasOptionalLabel", false),
+  hasRequiredLabel: boolean("hasRequiredLabel", false),
+  id: text("id", "input-id"),
```

Comment:
I don't think this is really necessary for a knob.  It's part of the API, but not a very visible one that can be shown off in the Showcase.